google-api-client/pom.xml

```diff
@@ -154,5 +159,17 @@
       <groupId>com.google.http-client</groupId>
       <artifactId>google-http-client-apache-v2</artifactId>
     </dependency>
+    <dependency>
```

Comment:
We use Mockito for all new tests

Reply:
I need to mock static method and environment variables, it seems Mockito cannot do either of these.

Reply:
Mockito can mock static methods as of version 3.4.

Reply:
To mock static method with Mockito, it seems we need to either pass in a function reference or a lambda to `when`, but none of these are supported in Java 7.

```
try(MockedStatic<Utils> utilsMock = Mockito.mockStatic(Utils.class)) {
      // use method reference inside "when"
      utilsMock.when(Utils::loadDefaultCertificate).thenReturn(mtlsKeyStore);
}
```
```
try (MockedStatic<System> systemMock = Mockito.mockStatic(System.class)) {
      // use lambda inside "when"
      systemMock.when(() -> System.getenv("foo")).thenReturn("true");
}
```

Reply:
It seems there is no way to use Mockito to mock static for Java 7 since Java 7 doesn't support method reference and lambda, so I will still use PowerMockito.